cmds/exp/disk_unlock/disk_unlock.go

```diff
@@ -35,13 +36,16 @@ const (
 	hostSecretSeedLen = 32
 
 	passwordSalt = "SKM PROD_V2 ACCESS"
+
+	SkmMPI = 0x0601
```

Comment:
what's this, and why is it exported

Reply:
ah thanks for catching that, it shouldnt be exported